extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-tag.js

```diff
@@ -14,9 +19,19 @@ describes.realWin(
     let win;
     let element;
     let shoppingTag;
+    let storeService;
+
+    const nextTick = () =>
+      new Promise((resolve) => win.setTimeout(resolve, 100));
```

Comment:
nextTick refers to other testing libraries with `tick` kind of helpers (or NodeJS). They run all the microtasks, i.e. all the work that was scheduled while running the code, and can be executed instantly (resolved promises, etc).
If you use a timeout of 100ms it's no longer a `tick` but a timeout.

Do you really need this random bit of 100ms async? It's really an anti pattern